packages/paste-libraries/data-visualization/stories/index.stories.tsx

```diff
@@ -0,0 +1,116 @@
+import * as React from 'react';
+import type {Story, Meta} from '@storybook/react';
+import {Stack} from '@twilio-paste/stack';
+/* eslint-disable import/no-extraneous-dependencies */
+import Highcharts from 'highcharts';
+import HighchartsReact from 'highcharts-react-official';
+/* eslint-enable */
+
+import {usePasteHighchartsTheme} from '../src';
+
+// eslint-disable-next-line import/no-default-export
+export default {
+  title: 'Libraries/data-visualization',
+} as Meta;
+
+const lineChartOptions: Highcharts.Options = {
+  title: {
+    text: 'Solar Employment Growth by Sector, 2010-2016',
+  },
+  subtitle: {
+    text: 'Source: thesolarfoundation.com',
+  },
+  series: [
+    {
+      name: 'Installation',
+      // @ts-expect-error copied from official highcharts docs
+      data: [43934, 52503, 57177, 69658, 97031, 119931, 137133, 154175],
```

Comment:
Looking at the beta docs for TS and Highcharts, they[ suggest casting the data as the correct type](https://www.highcharts.com/docs/advanced-chart-features/highcharts-typescript-declarations#debugging). Did you try that before adding the expect error statement?

Reply:
Yea I tried that first but kept getting a slew of other issues, even after doing `as unknown) as SeriesXrangeOptions` etc. 🤷‍♂️